### ISSUE TITLE
Navigation sections are rendered as links

### ISSUE THREAD

**Issue by lastsecondsave** [squidfunk/mkdocs-material#4358]
### Contribution guidelines

- [X] I've read the [contribution guidelines](https://github.com/squidfunk/mkdocs-material/blob/master/CONTRIBUTING.md) and wholeheartedly agree

### I've found a bug and checked that ...

- [X] ... the problem doesn't occur with the `mkdocs` or `readthedocs` themes
- [X] ... the problem persists when all overrides are removed, i.e. `custom_dir`, `extra_javascript` and `extra_css`
- [X] ... the documentation does not mention anything about my problem
- [X] ... there are no open or closed issues that are related to my problem

### Description

In the version 8.5.0 a first navigation section is rendered as a link:

![image](https://user-images.githubusercontent.com/6729879/190153229-7ef4427d-928c-43d3-acd6-2b554fdf5684.png)

### Expected behaviour

As it's just a title nothing should happen when you hover over it.

### Actual behaviour

When you hover over it, the section acts as a link, although it's not clickable. Other sections renders fine.

### Steps to reproduce

Enable features mentioned in the configuration below.

### Package versions

- Python: `3.10.4`
- MkDocs: `1.3.0`
- Material: `8.5.0`

### Configuration

```yaml
theme:
  name: material
  features:
    - navigation.sections
    - navigation.tabs
    - toc.integrate
```

### System information

- Browser: Firefox, Chrome

**Comment by squidfunk** [comment]
Thanks for reporting. Does this also happen with 8.4.4?

**Comment by lastsecondsave** [comment]
No, on 8.4.4 everything is good.

**Comment by squidfunk** [comment]
Okay, it's a regression then.

**Comment by squidfunk** [comment]
Fixed in f30775fa0. It was indeed a regression related to the merge, as the navigation code is slightly different in Insiders due to the support for [navigation icons](https://squidfunk.github.io/mkdocs-material/reference/#setting-the-page-icon), [status](https://squidfunk.github.io/mkdocs-material/reference/#setting-the-page-status) and [pruning](https://squidfunk.github.io/mkdocs-material/setup/setting-up-navigation/#navigation-pruning).

**Comment by squidfunk** [comment]
Released as part of 8.5.2.
